src/Kconfig

```diff
@@ -243,3 +243,19 @@ config GOLIOTH_DEBUG_DEFAULT_LOG_LEVEL
 
         This value can be overriden at runtime with golioth_debug_set_log_level().
 
+choice GOLIOTH_AUTH_METHOD
+    prompt "Authentication method support"
+
+config GOLIOTH_AUTH_METHOD_PSK
+    bool "PSK-ID / PSK only"
+    imply GOLIOTH_AUTH_PSK_MBEDTLS_DEPS
```

Comment:
This is defined only in Zephyr port, but I don't have a better way to actually pull those mbedTLS dependencies in another way.

Reply:
Hmm yeah, that's a good point. I'll noodle on this, we can make an improvement later.